src/installer-source/KolibriSetupScript.iss

```diff
@@ -495,34 +478,62 @@ end;
 const
     DEFAULT_PIP_PATH = '\Python34\Scripts\pip.exe';
 { Returns the path of pip.exe on the system. }
-{ Tries several different locations before prompting user. }
-
-
-function FailedPipNotFound() : String;
-begin
-    // MsgBox('FailedPipNotFound - ' + GetExceptionMessage, mbInformation, mb_Ok);
-    // Log('Exception: ' + GetExceptionMessage);
-    CustomizeMsgbox('File: ' + ExpandConstant('{sd}') + DEFAULT_PIP_PATH + CustomMessage('FileNotFound'));
-    RemoveOldInstallation(ExpandConstant('{app}'));
-    forceCancel := True
-    ExitProcess(1);
-    end;
- 
-function GetPipPath() : String;
-var
-    path : string;
-begin
-    path := ExpandConstant('{sd}') + DEFAULT_PIP_PATH;
-    if FileExists(path) then
-        begin
-            result := path;
-            exit;
-        end;
-    begin
-        FailedPipNotFound();
-        result := '';
-    end;
-end;
+{ Tries several different locations before prompting user. }
+
+function FailedPipNotFound() : String;
+begin
+    // MsgBox('FailedPipNotFound - ' + GetExceptionMessage, mbInformation, mb_Ok);
+    // Log('Exception: ' + GetExceptionMessage);
+    CustomizeMsgbox('File: ' + ExpandConstant('{sd}') + DEFAULT_PIP_PATH + CustomMessage('FileNotFound'));
+    RemoveOldInstallation(ExpandConstant('{app}'));
+    forceCancel := True
+    ExitProcess(1);
+end;

```

Comment:
do you know why the git diff on this file is so large? The text looks unchanged.

If there was some hidden change in endlines or whitespace, it would be better to do that in a separate commit to help see what has actually changed

Reply:
Previously, there were 2 linebreaks (line 499 and 500), now there's one.. but yeah that's not a very big difference :)

Reply:
And `GetDefaultPip` (below) is new..

Reply:
that wasn't quite what I was asking @benjaoming. I'm saying that the diff is showing changes even though the text is the same

Reply:
> do you know why the git diff on this file is so large? The text looks unchanged.

I think its because of the white space, I try to restore it.